app/services/stacks/unexpected_metadata_response.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# A custom exceptions class to provide specific Stacks::Exceptions
+module Stacks
+  # An UnexpectedMetadataResponse to raise when HTTP returns bad JSON
+  class UnexpectedMetadataResponse < StandardError
+    def initialize(url = nil, message = nil)
```

Comment:
When you call this you are passing a `ParserError` to the argument you are  calling `message`